addons/l10n_syscohada/data/account_tax_template_data.xml

```diff
@@ -61,10 +65,12 @@
         <field name="tax_group_id" ref="tax_group_0"/>
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {'repartition_type': 'base'}),
+
             (0,0, {'repartition_type': 'tax'}),
         ]"/>
         <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {'repartition_type': 'base'}),
+
```

Comment:
@malb-odoo Why?  What is up with the empty lines?

Reply:
The scripts did that i think, i will correct that since it's not needed

Reply:
@jco-odoo Done